docs/proposals/llm-hyperparameters-tuning-api.md

```diff
@@ -0,0 +1,135 @@
+# LLM Hyperparameters Tuning API Proposal
```

Comment:
Should we call it: HyperParameter Optimization API for LLM Fine-Tuning ?
Thoughts @kubeflow/wg-training-leads @deepanker13 @helenxie-bit ?

Reply:
Yes @andreyvelich , it sounds more descriptive.

Reply:
Thank you! People may tend to get confused about "hyperparameter tuning" and "fine-tuning", so I think this one is better. I have renamed the proposal with it.